node_modules/.yarn-integrity

```diff
@@ -0,0 +1,16 @@
+{
+  "systemParams": "darwin-x64-67",
+  "modulesFolders": [
```

Comment:
@brunomuler we should not check-in the node_modules directory. it will be wiped if we ever run the `./scripts/clean.sh` script. this should have been gitignored. can you remove and add to the gitignore list?